openedx/core/djangoapps/schedules/tasks.py

```diff
@@ -57,6 +65,7 @@ def __init__(self, day, *args, **kwargs):
         self.name = "recurringnudge_day{}".format(day)
 
 
+# TODO: delete once recurring_nudge_schedule_bin is fully rolled out
 @task(ignore_result=True, routing_key=ROUTING_KEY)
 def recurring_nudge_schedule_hour(
```

Comment:
What is this being used for currently? Are we just keeping it around because we will have overlapping celery queues during deploys?

Reply:
Yep, this is so that we can red-black deploy this. I added a task to the wiki to clean this up after this merges.